helm/values/prod-values.yaml

```diff
@@ -0,0 +1,39 @@
+name: callisto-timecard-restapi
+
+service:
+  name: callisto-timecard-restapi
+  port: 3000
+
+replicaCount: 1
```

Comment:
Not an immediate concern, but at some point we would need this number to increased to foster availability/scalability in production?

Reply:
Do you think we should have a ticket created or something to bear in mind?

Reply:
I think that would be a good idea. We can easily forget about it. The architecture team could then feed into that decision as well. Let me know once the ticket is created and I will solve this comment